torch/onnx/_internal/fx/export.py

```diff
@@ -415,6 +433,18 @@ def _export_fx_to_onnxscript(fx_module_with_metadata, opset_version):
             assert isinstance(output, (graph_building.TorchScriptTensor, tuple)), type(
                 output
             )
+            # op-level validation
+            # TODO(titaiwang): Change ORTEvaluator to ReferenceEvaluator
+            # Symbolic_fn should have the same output as node.target (torch ops)
+            with evaluator.default_as(evaluator.ORTEvaluator()):
```

Comment:
```suggestion
            with evaluator.default_as(evaluator.ort_evaluator):
```

They have some caching mechanism there